modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -2021,6 +2021,584 @@ warn: feature deprecated without replacement.
 ======
 =====
 
+.Using an unescaped variable named 'where' in node
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+
+Query::
++
+[source,cypher]
+----
+MATCH (where {p: 5})
+RETURN where
+----
+
+Description of the returned code::
+`(where {p: 5})` is deprecated. It is replaced by `(++`where`++ {p: 5})`.
+
+Suggestions for improvement::
+To continue using variables with this name, use backticks to escape the variable name:
++
+[source,cypher]
+----
+MATCH (`where` {p: 5})
+RETURN `where`.p
+----
+======
+
+[.include-with-GQLSTATUS-deprecated-with-replacement]
+======
+Query::
++
+[source,cypher]
+----
+MATCH (where {p: 5})
+RETURN where
+----
+
+Returned GQLSTATUS code::
+01N01
+
+Returned status description::
+warn: feature deprecated with replacement.
+`(where {p: 5})` is deprecated.
+It is replaced by `(++`where`++ {p: 5})`.
+
+Suggestions for improvement::
+To continue using variables with this name, use backticks to escape the variable name:
++
+[source,cypher]
+----
+MATCH (`where` {p: 5})
+RETURN `where`.p
+----
+======
+=====
+
+.Using an unescaped variable named 'where' in relationship patterns
```

Comment:
```suggestion
.Using an unescaped variable named 'where' in a relationship pattern
```
For consistency with the above suggestion. Plural vs. singular is your call. It just seems to me singular is used more on this page.